### DISCUSSION TITLE
Sidebery use with Simple Tab Groups issue

### DISCUSSION THREAD

**Discussion by gabebohlmann** [mbnuqw/sidebery#1453] [Q&A]
I would like to use Sidebery in conjunction with the Simple Tab Groups extension, but some functionality is broken. I would like Sideberry to only display the tabs open in the current simple tab group. It instead shows all tabs open across all simple tab groups. Tree Style Tab has this functionality, so it is possible. I understand that Sidebery is in some ways a competitor to Simple Tab Groups, but I think this functionality should be possible/enabled. My use case is that I have Sidebery disabled by default until I have too many tabs open and then I switch to Sidebery with the top native tab bar hidden with userChrome.css

**Comment by wsalopek** [comment]
I second this - it creates a way for Firefox to have vertical tabs AND "workspaces" (I think) via Simple Tab Groups.  It's extremely disappointing that Sidebery does not sync panels - if it did, Firefox/Sidebery would have syncable "workspaces".  Edge has workspaces that sync and work great, but I want to stay with Firefox, and with Sidebery.
